main/resource-access.bs

```diff
@@ -67,13 +69,35 @@ if it needs to access resources requiring authentication (see [[#webid]]).
 When it receives a response with a `403` or `404` status code,
 it MAY repeat the request with different credentials.
 
+A Solid client MUST use the `Content-Type` request header in `PUT`, `POST` and `PATCH` requests. [[Source](https://github.com/solid/specification/issues/70#issuecomment-547924171)]
 
-## Linked Data Platform ## {#ldp}
+## Uniform Resource Identifier ## {#uri}
 
-Issue: Write Linked Data Platform section.
+### Required shared slash semantics ### {#uri-path-component}
 
-Draft:
-A Solid data pod MUST conform to the LDP specification [[!LDP]].
+Server and client share the slash semantics in the URI path component representing hierarchy [[!RFC3986]]. [[Source](https://github.com/solid/specification/issues/35#issuecomment-547949014)]
+
+Server MUST NOT create a URI differing only in the trailing slash with that of an existing URI. [[Source](https://github.com/solid/specification/issues/107#issuecomment-567482817)]
```

Comment:
Servers don't create URIs; URIs always exist. What exactly do we mean? That a redirect is needed? That either `foo` or `foo/` should yield a 404?

Reply:
A URI doesn't "exist" until it is named, registered, or associated with a resource... but I get your point, will update. Yes, a redirect in the following line.

Reply:
> A URI doesn't "exist" until it is named, registered, or associated with a resource...

It does: https://csarven.ca/it-does
A URI is a string conforming to a certain pattern. The set of all URIs that will ever be minted exists, it just happens to be infinite.

So everything needs to be phrased in terms of resources being created.

Reply:
Not sure how deep or philosophical (mathematical?) you want to go, but I hope we can hover on being pragmatic. Sure, I get it, certainly a set of all things exists on a plane but not sure how you want to put that knowledge into actual use. We are basing things off RFC, WebArch.. have I overlooked your exists or can't recall a DesignIssue?

URIs exist to identify a resource. Server advertises a name because it happened to associate a resource with a URI - I used "create" as a generally comprehensible terminology. I'm okay to change that though.

Reply:
No philosophy indeed, pragmatic but exact.

As I wrote above, it’s a resource that is being created on the server; that’s the state change that happens and that we should be describing across the entire document.

It’s important to be exact, as URIs can be minted before the corresponding resource is created. Common case in point: an ACL URL for a specific resource (while that resource currently inherits another ACL).

Reply:
The ACL example is precisely what I mean by naming. The URI starts to exists because it is named / written down. That's being pragmatic. There is an artifact, the string, spelled out, it is visible.. we can talk about it. On the other hand, what's the point in saying that something somehow exists but not actually name it? That'd be being philosophical. Like I said, the context of "create" was because - and I'll use you own words - "a resource is **created** on the server" and that it is identified with a URI.

I'll revise tomorrow.

Reply:
Got it. Suggestion:

"If two URIs differ only in the trailing slash, and the server has associated a resource with one of them, then the other URI MUST NOT correspond to another resource. Instead, the server MAY respond to requests for the latter URI with a 301 redirect to the former. Behaviour pertaining to authorization MUST proceed this optional redirect."

Reply:
https://github.com/solid/specification/commit/44a0c2d9408be26fe44a82bd0a0a7a25378ce838